dlls/ff/ff_player.cpp

```diff
@@ -1708,15 +1666,10 @@ void CFFPlayer::SetupClassVariables()
 	}
 
 	ClearSpeedEffects();
-
-	// clear recent attacker (kill assist) info
-	m_recentAttackers.Purge();

```

Comment:
This looks relevant on first glance--why is it removed here?

Reply:
Rebase conflict accident I suspect

Reply:
Not sure why this isn't showing as outdated...